lib/esp.rb

```diff
@@ -1,71 +1,92 @@
 module ESP
   # Manually set the access_key_id you created from https://esp.evident.io/settings/api_keys.
   #
-  # You can optionally set the `ESP_ACCESS_KEY_ID` environment variable.
+  # You can optionally set the +ESP_ACCESS_KEY_ID+ environment variable.
+  #
+  # @param access_key_id [String] Your access key ID.
+  # @return [void]
   def self.access_key_id=(access_key_id)
     @access_key_id               = access_key_id
     ESP::Resource.hmac_access_id = access_key_id
   end
 
-  # Reads the `ESP_ACCESS_KEY_ID` environment variable if ::access_key_id was not set manually.
+  # Reads the +ESP_ACCESS_KEY_ID+ environment variable if {ESP#access_key_id= ::access_key_id} was not set manually.
```

Comment:
I don't think we need the `::`.  I think that is the way to reference internal links with rdoc.  These can all just be `{.access_key_id}` for class methods, and `{#access_id}` if it were an instance method.